common/misc.ml

```diff
@@ -1096,6 +1102,7 @@ let ASSEMBLER_SIMPLIFY_TAC =
     DEPTH_CONV
      (GEN_REWRITE_CONV I [BITVAL_CLAUSES] ORELSEC
       WORD_RED_CONV ORELSEC
+      apply_extra_word_convs ORELSEC
```

Comment:
Oh - wait a second, since `apply_extra_word_convs` will effectively succeed everytime, and we are using ORELSEC here, does it imply that ` NORMALIZE_ADD_SUBTRACT_WORD_CONV ORELSEC NUM_RED_CONV ORELSEC INT_RED_CONV` won't run at all?

Reply:
Nevermind, `FIRST_CONV` fails if all of the conversions fail. I thought that `FIRST_CONV` should always succeed.